CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Change Log
 
+## 0.8.0 - 2025-01-22
+Summary of major changes:
+1. Managed identity support has been added for connecting container apps to container registries. Assuming you have sufficient privileges, this is the preferred method for securing these resources.
+1. A new `Container` view has been added along with a host of new commands, providing more ways to edit container images and environment variables directly
```

Comment:
What do you think about adding some screenshots for these two changes? I think it could be helpful for users to see it visually.